pandas/core/indexes/datetimes.py

```diff
@@ -1964,18 +1967,33 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
 
             .. versionadded:: 0.19.0
 
-        infer_dst : boolean, default False
-            .. deprecated:: 0.15.0
-               Attempt to infer fall dst-transition hours based on order
 
         Returns
         -------
         localized : DatetimeIndex
 
+	Examples
+    	--------
+    	In the example below, We put the date range from 01 March 2018 to 08 March 2018
+	& convert this to US/Eastern Time zone
+ 
+    	>>> import pandas as pd
```

Comment:
The import of pandas is not needed (check https://python-sprints.github.io/pandas/guide/pandas_docstring.html#conventions-for-the-examples)